github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```diff
@@ -465,6 +465,7 @@ periodics:
       - --config-path=github/ci/prow-deploy/files/orgs.yaml
```

Comment:
Should this config path be removed as this file will no longer exist?

Reply:
Oh, nice catch, thanks! Rebasing seems to have gone wrong, will fix it.